privacy-on-beam/pbeam/count.go

```diff
@@ -125,47 +120,30 @@ func Count(s beam.Scope, pcol PrivatePCollection, params CountParams) beam.PColl
 		newDecodePairInt64Fn(partitionT.Type()),
 		countPairs,
 		beam.TypeDefinition{Var: beam.XType, T: partitionT.Type()})
-
-	if params.specifiedPartitions.partitionsSpecified { // Partitions specified.
-		return addSpecifiedPartitionsForCount(s, epsilon, delta, maxPartitionsContributed,
-			params, noiseKind, countsKV)
-
+	// Add specified partitions, if partitions are specified.
```

Comment:
Hmm, since we are doing more than adding specified partitions here, maybe we could rename this comment to something like "Add specified partitions and return the aggregation output, if partitions are specified."

(Also for other aggregations)

Reply:
Good point